### ISSUE TITLE
Developing vs Contributing?

### ISSUE THREAD

**Issue by nikhiltri** [jehna/readme-best-practices#5]
What's the difference between what should be included in the "Developing" section versus the "Contributing" section? Aren't those essentially the same things?

**Comment by jehna** [comment]
You're right that those two concepts are now a bit overlapping with each other.

While "Developing" is meant to be a collection of technical steps you need to take in order to fiddle with the project, "Contributing" is meant to be a paragraph that lowers the bar for new contributors - a friendly advertisement to welcome people to contribute.

How do you see we could make the headlines more clear?

**Comment by nikhiltri** [comment]
The sentiment of the Contributing section sounds like it would be a nice intro to the content of the Developing section. It my [latest project](https://github.com/aic-collections/aic-gridview), I included everything in the Contributing section, and tried to make it clear that changes are welcome!
